lib/mongo_ecto/change.ex

```diff
@@ -11,7 +11,7 @@ defmodule Mongo.Ecto.ChangeMap do
   @behaviour Ecto.Type
 
   defstruct [:field, :value]
-  @type t :: %__MODULE__{field: String.t, value: term}
+  @type t :: %__MODULE__{field: String.t(), value: term}
```

Comment:
It would also be nice if we didn't require parens around type specs

Reply:
Not sure if it's possible. Looks like Elixir keep the parens CF: https://github.com/elixir-lang/elixir/blob/master/lib/elixir/lib/calendar.ex#L65

Reply:
Okay. I can live with it. Thanks!